plugins/modules/win_unzip.ps1

```diff
@@ -144,11 +145,18 @@ If ($ext -eq ".zip" -And $recurse -eq $false) {
         Fail-Json $result "Error importing module PSCX"
     }
 
-    Try {
-        Expand-Archive -Path $src -OutputPath $dest -Force -WhatIf:$check_mode
-    } Catch {
-        Fail-Json -obj $result -message "Error expanding '$src' to '$dest'! Msg: $($_.Exception.Message)"
-    }
+	Try{
+		If ($null -ne $password){
+                        $password= ConvertTo-SecureString $password -AsPlainText -Force
+                        Expand-Archive -Path $src -OutputPath $dest -Password $password -WhatIf:$check_mode
+                   }
+                Else{
+                        Expand-Archive -Path $src -OutputPath $dest -Force -WhatIf:$check_mode
+              }
+	}
+	Catch{
+ 		Fail-Json -obj $result -message "Error expanding '$src' to '$dest'! Msg: $($_.Exception.Message)"
+	     }
```

Comment:
You can use splatting to your advantage here and only have to call `Expand-Archive` once. Also try to be consistent with the existing indent spacing and style that exists in the module.

```suggestion
    $expand_params = @{
        Path = $src
        OutputPath = $dest
        WhatIf = $check_mode
    }
    if ($null -ne $password) {
        $expand_params.Password = ConvertTo-SecureString -String $password -AsPlainText -Force
    }
    Try {
        Expand-Archive @expand_params
    }
    Catch {
        Fail-Json -obj $result -message "Error expanding '$src' to '$dest'! Msg: $($_.Exception.Message)"
    }
```